teleport.js

```diff
@@ -1,11 +1,9 @@
 /**
  * wrapper to cleanly inject the code and pass the gameSpace context to the developer
+ * we keep the wrapper for future breaking updates
  */
 function wrapper(fn) {
-  const tmp = gameSpace.__proto__.keyE
-  gameSpace.__proto__.keyE = function() { fn(this) }
-  document.body.dispatchEvent(new KeyboardEvent('keydown', { 'key': 'e' }))
-  gameSpace.__proto__.keyE = tmp
+  fn(gameSpace);
```

Comment:
i address that in the comment beforehand, since the api is likely to change, keeping the wrapper function will allow us to do any transformations we need to fetch/get/rebuild the gameSpace object in the future